test/Lava.spec.ts

```diff
@@ -14,15 +18,20 @@ describe('Lava', () => {
     });
 
     beforeEach(function () {
-        scene.onTestStart(() => {});
+        scene.onTestStart((_lava, _player) => {
+            lava = _lava;
+            playerMock = _player;
+        });
         phaserTester.onTestStart(this.currentTest.title);
     });
 
-    it('should move up', () => {
-        assert.fail('Not implemented');
+    it('should move up', async () => {
+        const oldLavaY: number = lava.y;
+        await phaserTester.delay(1000);
+        expect(lava.y).to.be.lessThan(oldLavaY);
     });
 
-    it('should be able to speed up over time', () => {
+    it('should be able to speed up after 10 seconds', () => {
```

Comment:
- [ ] Add more tests that clearly define the original intended behaviour of lava
- [ ] Move functionality of lava away from `MainScene` and into `Lava` so that it can be tested as well